src/zenml/orchestrators/step_launcher.py

```diff
@@ -186,6 +189,28 @@ def launch(self) -> None:
             publish_utils.publish_failed_pipeline_run(pipeline_run.id)
             raise
 
+    def _get_step_docstring_and_source_code(self) -> Tuple[Optional[str], str]:
+        """Gets the docstring and source code of the step.
+
+        If any of the two is longer than 1000 characters, it will be truncated.
+
+        Returns:
+            The docstring and source code of the step.
+        """
+        from zenml.steps.base_step import BaseStep
+
+        step_instance = BaseStep.load_from_source(self._step.spec.source)
+
+        docstring = step_instance.docstring
+        if docstring and len(docstring) > 1000:
```

Comment:
Shouldn't this be `TEXT_FIELD_MAX_LENGTH<3` or something? 1000 seems quite a low amount of characters for code/docstring?

Reply:
I thought about that but intentionally kept it at 1000. The use case of this is only to have something to display in the UI, I don't think it's worth potentially saving 65KB of data per step for that